tests/converter_op_test.py

```diff
@@ -3455,6 +3455,38 @@ def forward(self, x):
         tfl_output = tfl_run_model(model_path, dummy_input, dummy_output)
         assert_close(dummy_output, tfl_output)
 
+    @unittest.skipIf(not hasattr(torch.nn.quantized, 'Conv2d'), 'Quantized conv2d is not supported')
```

Comment:
It seems that you forgot to remove this function, right?

Reply:
Thanks for your kindly reminder, pls kindly check the revision.